scripts/deploy.ts

```diff
@@ -1,15 +1,18 @@
 import {ethers} from "hardhat";
 
 async function main() {
+  const MinValidatorCount = 1;
+  //Max uint32
```

Comment:
Nitpick, please add a space between `//` and the rest of the comment

Reply:
Fixed 115588a7e5c6265e64aa2ce6755bf45a100913b9